src/component/RegisterForm/RegisterForm.js

```diff
@@ -0,0 +1,41 @@
+import React, {Component} from 'react';
+import firebase from '../firebase/firebase';
+
+export default class RegisterForm extends Component {
```

Comment:
Also, when I wanted to test it out locally, this component wasn't added to App.js and therefore it hasn't rendered. Would you delete the create-react-app content in App.js and add there your form?

Reply:
I  added the RegisterForm to the App.js file